CHANGELOG/CHANGELOG-4.1.md

```diff
@@ -0,0 +1,281 @@
+# Release notes for v4.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v4.0.0
+
+### Uncategorized
+
+- Kube client dependencies are updated to v1.26.0.
```

Comment:
Can you add 
`([#PR number](https://github.com/kubernetes-csi/external-attacher/pull/<xxxx>), [@PR author git id](https://github.com/git id))`?

Reply:
Fixed